kyma-project/serverless
# Add basic (CRUD) serverless operations to main test scenario

**Description**

Changes proposed in this pull request:

- test scenario:
  - create Serverless CR and verify result
  - upgrade Serverless CR with custom spec and verify result
  - delete Serverless CR and verify result

**Related issue(s)**

- https://github.com/kyma-project/serverless/issues/848#issue-2222275134
